.fixtures.yml

```diff
@@ -5,6 +5,6 @@ fixtures:
       ref: "4.6.0"
     apt:
       repo: "puppetlabs-apt"
-      ref: "2.0.0"
+      ref: "2.2.0"
```

Comment:
I don't believe our dependencies have any bearing on semver progression. If this change requires a bump, we bump. Updates of our version are just determined by the nature of the change itself.